test/PostHogTest.php

```diff
@@ -64,6 +73,21 @@ public function testIdentify(): void
         );
     }
 
+    public function testIsFeatureEnabled()
+    {
+        $this->assertFalse(PostHog::isFeatureEnabled('having_fun', 'user-id'));
+    }
+
+    public function testIsFeatureEnabledWithSimpleFlag()
+    {
+        $this->assertTrue(PostHog::isFeatureEnabled('simpleFlag', 'user-id'));
+    }
+
+    public function testFetchEnabledFeatureFlags()
+    {
+        $this->assertIsArray(PostHog::fetchEnabledFeatureFlags('user-id'));
+    }
+
```

Comment:
could we add a test for `isSimpleFlagEnabled`? 

Flag of key `a` with distinct id `b` should be on for rollout percentage `42` but off for rollout percentage `40`